test/metabase/driver/postgres_test.clj

```diff
@@ -7,9 +7,7 @@
              [query-processor-test :refer [rows]]
              [sync :as sync]
              [util :as u]]
-            [metabase.driver
-             [generic-sql :as sql]
-             postgres]
```

Comment:
`cljr-clean-namespace` probably did this for you automatically but the reason we're loading the postgres driver here is that the `import` statement below it will fail if some other namespace hasn't already loaded the postgres driver. Thus something like `lein test metabase.driver.postgres-test` could potentially fail.

It's unfortunate that there's no way to tell `cljr-clean-ns` that some of those lines are there on purpose like that. But I would just keep that in mind when you review your own PR, if there's a required namespace that doesn't look like it's being used for something being removed take a look at the `:import`s as well